### PR TITLE
Adjust torch BKC version

### PR DESCRIPTION
**Pull request by vshampor**

### CODE REVIEW

File: tests/onnx/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 pytest-mock>=3.3.1
 pytest-dependency>=0.5.1
 pytest-ordering>=0.6
-torch==1.13.0
-torchvision==0.14.0
+torch~=1.13.0
```
Comment (ljaljushkin) [inline tests/onnx/requirements.txt:4]: @kshpv please take a look

**Review by ljaljushkin — APPROVED** [review]

File: tests/onnx/requirements.txt (hunk 1 above)
Comment (kshpv) [inline tests/onnx/requirements.txt:4]: @vshampor why did you update torch version in tests? What if any bugs will be added in a new release version of torch? Moreover if they update the models inside torchvision?
I would prefer to keep it strict (==)

**Review by kshpv — CHANGES_REQUESTED** [review]

File: tests/onnx/requirements.txt (hunk 1 above)
Comment (vshampor) [reply tests/onnx/requirements.txt:4]: Patch version updates tend to fix bugs, not add them.

Comment (kshpv) [reply tests/onnx/requirements.txt:4]: Ok, since the 1.13 has the security issues let's update it.

**Review by kshpv — APPROVED** [review]

**Pull request merged**